src/AddDiveWish.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Button from './Button.js';
+
+export default function AddListItem({ onAddDiveWish }) {
+  return (
+    <Form
+      aria-label="Add a dive destination to your wishlist"
+      onSubmit={handleAddDiveWish}
+    >
+      <Container>
+        <Label htmlFor="destination">DESTINATION </Label>
+        <Input id="destination" name="destination" required />
+      </Container>
+      <Container>
+        <Label htmlFor="notes">NOTES</Label>
+        <Textarea id="notes" name="notes" required />
+      </Container>
+      <Button type="submit">Add to list</Button>
+    </Form>
+  );
+
+  function handleAddDiveWish(event) {
+    event.preventDefault();
+    const form = event.target;
+    const input = form.elements.destination;
+    const textarea = form.elements.notes;
+    onAddDiveWish(input.value, textarea.value);
```

Comment:
Just a thought: maybe it would be easier to further work with the data if you'd already pass it as an object?
```suggestion
    onAddDiveWish({ destination: input.value, notes: textarea.value});
```

Reply:
Thanks, I didn't think about that!